systemtest/src/test/java/io/strimzi/systemtest/upgrade/StrimziUpgradeIsolatedST.java

```diff
@@ -63,8 +64,7 @@ void testUpgradeStrimziVersion(String fromVersion, String toVersion, String fgBe
 
     @Test
     void testUpgradeKafkaWithoutVersion(ExtensionContext extensionContext) throws IOException {
-        UpgradeDowngradeDatalist upgradeDatalist = new UpgradeDowngradeDatalist();
-        UpgradeDowngradeData acrossUpgradeData = upgradeDatalist.buildDataForUpgradeAcrossVersions();
+        BundleVersionModificationData acrossUpgradeData = new VersionModificationDataLoader(ModificationType.BUNDLE_UPGRADE).buildDataForUpgradeAcrossVersions();
```

Comment:
When you will have `VersionModificationDataLoader` as singleton, you will not load the data every time.